src/main/resources/io/jenkins/plugins/audit/config/AuditLogConfiguration/config.jelly

```diff
@@ -5,7 +5,15 @@
     <f:entry title="${%Log destination}" field="logDestination">
         <f:textbox name="logDestination" value="${it.logDestination}"/>
     </f:entry>
-
+    <f:entry title="${%IANA Enterprise Number}" field="enterpriseNumber">
+        <f:textbox name="enterpriseNumber" value="${it.enterpriseNumber}"/>
+    </f:entry>
+    <f:entry name="appenderType" title="Choose the Log Layout" field="appenderType">
```

Comment:
```suggestion
    <f:entry name="appenderType" title="${%Log Appender Type}" field="appenderType">
```

For one, we should make UI strings localizable like that. For another, I think it would be more accurate to label this as the appender type more so than the output format (i.e., layout). We can probably make the layout a sub-configuration essentially for each appender type. With that in mind, it might be useful to try out one of the hetero-list jelly UI components for this instead of a select list. See https://github.com/jenkinsci/ui-samples-plugin for some samples of writing more complex jelly forms.

Reply:
Also, the `appenderType` field needs to be added to `AuditLogConfiguration` with a getter and setter similar to `logDestination`.